content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -1,5 +1,278 @@
 ---
-title: "Pods"
+reviewers:
+- erictune
+title: Pods
+content_type: concept
 weight: 10
+no_list: true
+card:
+  name: concepts
+  weight: 60
 ---
 
+<!-- overview -->
+
+_Pods_ are the smallest deployable units of computing that can be created and
+managed in Kubernetes.
+
+A _Pod_ (as in a pod of whales or pea pod) is a group of one or more
+{{< glossary_tooltip text="containers" term_id="container" >}} (such as
+Docker containers), with shared storage/network, and a specification
+for how to run the containers.  A Pod's contents are always co-located and
+co-scheduled, and run in a shared context.  A Pod models an
+application-specific "logical host"&mdash;it contains one or more application
+containers which are relatively tightly coupled&mdash;in a pre-container
+world, being executed on the same physical or virtual machine would mean being
+executed on the same logical host.
+
+As well as application containers, a Pod can contain
+[init containers](/docs/concepts/workloads/pods/init-containers/) that run
+during Pod startup. You can also inject
+[ephemeral containers](/docs/concepts/workloads/pods/ephemeral-containers/)
+for debugging if your cluster offers this.
+
+<!-- body -->
+
+## What is a Pod?
+
+{{< note >}}
+While Kubernetes supports more
+{{< glossary_tooltip text="container runtimes" term_id="container-runtime" >}}
+than just Docker, [Docker](https://www.docker.com/) is the most commonly known
+runtime, and it helps to describe Pods using some terminology from Docker.
+{{< /note >}}
+
+The shared context of a Pod is a set of Linux namespaces, cgroups, and
+potentially other facets of isolation - the same things that isolate a Docker
+container.  Within a Pod's context, the individual applications may have
+further sub-isolations applied.
+
+In terms of Docker concepts, a Pod is similar to a group of Docker containers
+with shared namespaces and shared filesystem volumes.
+
+## Using Pods
+
+Usually you dont' need to create Pods directly. Instead, you can almost
+always use workload resources such as {{< glossary_tooltip text="Deployment"
+term_id="deployment" >}} or {{< glossary_tooltip text="Job" term_id="job" >}},
+even for singleton Pods.
+If your Pods need to track state, there is a
+{{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}} resource intended
+to be the best fit for that use case.
+
+Pods in a Kubernetes cluster can be used in two main ways:
+
+* **Pods that run a single container**. The "one-container-per-Pod" model is the
+  most common Kubernetes use case; in this case, you can think of a Pod as a
+  wrapper around a single container; Kubernetes manages Pods rather than managing
+  the containers directly.
```

Comment:
```suggestion
* **Pods that run a single container**. The "one-container-per-Pod" model is the
  most common Kubernetes use case. Think of a Pod as a
  wrapper around a single container; Kubernetes manages the Pod instead of managing
  the containers directly.
```